basis/common-router/src/main/java/com/softgroup/common/router/api/AbstractRequestHandler.java

```diff
@@ -18,14 +18,14 @@
     @Autowired
     protected HandlerFactory<RequestHandler> handlerFactory;
 
-    private Class<T> clazz;
+    private static Class clazz;
 
     @Override
     public Response<?> handle(Request<?> msg) {
-        clazz = (Class<T>) ((ParameterizedType) this.getClass().getGenericSuperclass()).getActualTypeArguments()[0];
+        clazz = (Class) ((ParameterizedType) this.getClass().getGenericSuperclass()).getActualTypeArguments()[0];

```

Comment:
Do you mean assigning value in the place of declaration? Like this?
private static Class clazz = (Class) ((ParameterizedType) this.getClass().getGenericSuperclass()).getActualTypeArguments()[0];